### PR TITLE
Added yml for Italian.

### PR DESCRIPTION
**Pull request by gianpaolof**
Added yml for Italian.
Fixed two typos in the English yml

### CODE REVIEW

File: Source/EyesGuard.Data/Languages/it-IT.yml [hunk 1]
```diff
@@ -0,0 +1,238 @@
+﻿---
+Meta:
+  Translators:
+  - Name: Aryan Ebrahimpour
```
Comment (0xaryan) [inline Source/EyesGuard.Data/Languages/it-IT.yml]: Please change translator info to your name

**Comment by gianpaolof** [comment]
Ok

File: Source/EyesGuard.Data/Languages/it-IT.yml (hunk 1 above)
Comment (gianpaolof) [reply Source/EyesGuard.Data/Languages/it-IT.yml]: Done

**Comment by 0xaryan** [comment]
@gianpaolof F# Yaml type provider inferred the type of Website as System.Uri due to base yml definition (en-US.yml). I set website to your github profile for now.

**Comment by gianpaolof** [comment]
Ah, that's ok. :-)

**Comment by 0xaryan** [comment]
Thanks for the contribution, I've also added you to English translators as you fixed typos.

image

**Comment by gianpaolof** [comment]
oh, very cool :-)

**Pull request merged**